draft-ietf-oauth-transaction-tokens.md

```diff
@@ -622,8 +626,8 @@ When creating Txn-Tokens, the Txn-Token MUST NOT contain the Access Token presen
 ## Subject Token Types {#sec-sub-token-types}
 A service requesting a Txn-Token SHOULD provide an incoming token if it has one that it used itself to authorize a caller, and if it directly correlates with the downstream call chain it needs the Txn-Token for. In the absence of an appropriate incoming token, the requesting service MAY use a self-signed JWT, an unsigned JSON object or any other format to represent the details of the requester to the Txn-Token service.
 
-## Client Authentication
-If using the `actor_token` and `actor_token_type` parameters of the OAuth 2.0 Token Exchange specification, both parameters MUST be present in the request. The `actor_token` MUST authenticate the identity of the requesting workload.
+## Use of 'actor_token'
+If using the `actor_token` and `actor_token_type` parameters of the OAuth 2.0 Token Exchange specification {{RFC8693}}, both parameters MUST be present in the request. The `actor_token` MUST authenticate the identity of the requesting workload.
```

Comment:
does it have to? wouldn't this preclude the `actor_token` being used in conjunction with some other form of authentication? 

```suggestion
If using the `actor_token` and `actor_token_type` parameters of the OAuth 2.0 Token Exchange specification {{RFC8693}}, both parameters MUST be present in the request. The `actor_token` can authenticate the identity of the requesting workload.
```